src/plugins/workspace/public/components/workspace_creator/workspace_permission_setting_panel.tsx

```diff
@@ -364,6 +364,31 @@ export const WorkspacePermissionSettingPanel = ({
     onChange?.([...userPermissionSettings, ...groupPermissionSettings]);
   }, [onChange, userPermissionSettings, groupPermissionSettings]);
 
+  const nonDeletableIndex = useMemo(() => {
```

Comment:
How about refactor code like these?
```
const userAdminIndex = userPermissionSettings.findIndex((permission)=>getPermissionModeId(permission.modes ?? []) === PermissionModeId.Admin);
const groupAdminIndex = groupPermissionSettings.findIndex((permission)=>getPermissionModeId(permission.modes ?? []) === PermissionModeId.Admin);

return {
        userNonDeletableIndex: userAdminIndex !== -1 && groupAdminIndex === -1 ? userAdminIndex : -1,
        groupNonDeletableIndex: userAdminIndex === -1 && groupAdminIndex !== -1 ? groupAdminIndex : -1
    };
```

Reply:
When there is two admin in users and not admin in group, user admin index is not -1 and group admin index is -1, the expected `userNonDeletableIndex` should be -1 instead of `userAdminIndex`.